code-challenges.js

```diff
@@ -31,24 +52,60 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+const bpMeasure = (num) => {
+if (num < 212) {
+return `Tempurature ${num} is below boiling point`
+}
+else if (num > 212) {
+    return `Temperature ${num} is above boiling point`
+}
+else {
+    (num === 212) 
+    return `Temperature ${num} is at boiling point`
+}
+}
+
+console.log(bpMeasure(temperature1))
+console.log(bpMeasure(temperature2))
+console.log(bpMeasure(temperature3))
+// Outcome: 
+// Tempurature 42 is below boiling point
+// Temperature 350 is above boiling point
+// Temperature 212 is at boiling point
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+// Input: The two provided arrays
+// Output: A number which is the length of the two arrays combined
+// Process: Use the + operator to combine the two arrays and use .length to measure the length
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+console.log(padres1984WorldSeriesRuns.length + (padres1998WorldSeriesRuns.length))
```

Comment:
When implemented like this, the two arrays are never getting combined! Since `.length` returns a number, you are actually just performing math with the two lengths of the arrays.